Userland/Libraries/LibJS/Bytecode/Interpreter.cpp

```diff
@@ -330,7 +330,14 @@ Interpreter::HandleExceptionResponse Interpreter::handle_exception(size_t& progr
     VERIFY_NOT_REACHED();
 }
 
-void Interpreter::run_bytecode(size_t entry_point)
+// FIXME: GCC takes a *long* time to compile with flattening, and it will time out our CI. :|
+#if defined(AK_COMPILER_CLANG)
```

Comment:
Is GCC slow in general, or just with sanitizers enabled? If (hopefully) the latter, we could just detect GCC+ASAN here

Reply:
Slow in general, it takes several minutes to build Interpreter.cpp on my machine, so I am reasonably sure it will kill CI :(